### PR TITLE
Update kaira

### PR DESCRIPTION
**Pull request by lumikani**
`kaira-core` no longer worked out of the box with a simple virtualenv setup and `pip install -r requirements.txt`. I updated some dependencies and did some clean up and checked that everything works with Python 3.9. Some issues still remain, for example `invoke regex-test` seems to only find a fraction of the matches it should for any given regex. But the output `kaira-core` produces when actually running an extraction of a testset is identical to the output produced by a run in 2019, and most tests pass. Failing tests are related to the NLP setup, which is going to be partially redone anyway, so they are marked as skipped tests for now.

### CODE REVIEW

File: tasks/__init__.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ def test(ctx):
     """
     Run tests
     """
-    ctx.run('python -m pytest')
+    ctx.run('python -m pytest --ignore .direnv')
```
Comment (Tumetsu) [inline tasks/__init__.py:12]: Tän fixupin vois varmaan vaan squashata tuohon alkuperäiseen nix setup committiin

File: flake.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (Tumetsu) [inline flake.nix:1]: Ehkä tää on nixistille selkee näinkin, mut oisko hyvä ajatus laittaa readmehen jotain ohjetta et btw, voit asentaa ympäristön yms. nixillä näin: lorem ipsum dolor..?

**Review by Tumetsu — APPROVED** [review]

File: tasks/__init__.py (hunk 1 above)
Comment (lumikani) [reply tasks/__init__.py:12]: Joo, mun workflow nykyään on et annan kaikkien fixuppien yms olla semmosenaan kunnes merge-aika. Ellen tee jotain välisiivouksia. Toki vois harkita et tekis fixupit ennenku submittaa PR:n :thinking: Se ois katselmoijille ehkä tavallaan kivempi.

File: flake.nix (hunk 2 above)
Comment (lumikani) [reply flake.nix:1]: done

File: tasks/__init__.py (hunk 1 above)
Comment (lumikani) [reply tasks/__init__.py:12]: Done

**Pull request merged**